packages/components/.storybook/preview.ts

```diff
@@ -0,0 +1,22 @@
+import type { Preview } from '@storybook/react';
+import DocumentationTemplate from './DocumentationTemplate.mdx';
+const preview: Preview = {
+
+    parameters: {
+        docs: {
+            page: DocumentationTemplate,
```

Comment:
we don't need the template

Reply:
I believe it makes sense to customize the template that is used when no custom documentation file is placed. If we define a custom file this wil not be generated.

Reply:
cool, no objection from my sid then 👍🏻